### ISSUE TITLE
Add documentation site.

### ISSUE THREAD

**Issue by bordoley** [bordoley/Reactive.re#8]
https://jaredforsyth.com/docre/index.html

**Comment by justgage** [comment]
Yes please, I think this library looks awesome however I don't know how to use it 🙈

**Comment by bordoley** [comment]
This is definitely on my todo prior to doing a proper release.

**Comment by arecvlohe** [comment]
Now moved to redoc: https://github.com/jaredly/redoc
